### ISSUE TITLE
RESOURCES

### ISSUE THREAD

**Issue by dupreesi** [yalla-coop/connect5#161]
Layers of C5
[layers .docx](https://github.com/yalla-coop/connect5/files/3157107/layers.docx)

List of local leads
[leads.docx](https://github.com/yalla-coop/connect5/files/3157109/leads.docx)

Train the trainers feedback forms: these are likely to get changed this Friday
[TtT.feedback.25.6.pdf](https://github.com/yalla-coop/connect5/files/3157112/TtT.feedback.25.6.pdf)
[S5.TtT.Evaluation.Form.amended.docx](https://github.com/yalla-coop/connect5/files/3157113/S5.TtT.Evaluation.Form.amended.docx)

PRE Course Survey
[Pre-course SurveyMonkey_151565181.pdf](https://github.com/yalla-coop/connect5/files/3157551/Pre-course.SurveyMonkey_151565181.pdf)

C5 Evaluation Questions
https://docs.google.com/spreadsheets/d/1_b_FhwKMmhh4wQpyU0i8vRh3b5cK37TvT3E5mq3FiUY/edit#gid=0

C5 Demographic Questions
https://docs.google.com/document/d/1ZSPrqGqbrL8FdVI7LjIKpfKFkI1BWyZPldwwzGYRv5Q/edit?usp=sharing

Suggestion for data visualisation from university
[Data visualisation behaviour change.docx](https://github.com/yalla-coop/connect5/files/3420996/Data.visualisation.behaviour.change.docx)

PARTICIPANT SESSION RESOURCES

SESSIONS 1 
http://online.anyflip.com/qage/giec/mobile/index.html
[session 1 self help resources .docx](https://github.com/yalla-coop/connect5/files/3422377/session.1.self.help.resources.docx)
[Connect 5 evaluation report.pdf](https://github.com/yalla-coop/connect5/files/3422382/Connect.5.evaluation.report.pdf)

SESSIONS 2
http://online.anyflip.com/qage/xhwd/mobile/index.html
[Connect5 suicide prevention resourcres .docx](https://github.com/yalla-coop/connect5/files/3422378/Connect5.suicide.prevention.resourcres.docx)

SESSIONS 3
http://online.anyflip.com/qage/vuzc/mobile/index.html
http://online.anyflip.com/qage/apij/mobile/index.html

TT
[Train the Trainer further reading.docx](https://github.com/yalla-coop/connect5/files/3422369/Train.the.Trainer.further.reading.docx)

User Manual 
[C5_Handbook_v1.pdf](https://github.com/yalla-coop/connect5/files/3451290/C5_Handbook_v1.pdf)

Elysabeth User Feedback 08.08.2019
[App feedback 8.docx](https://github.com/yalla-coop/connect5/files/3485982/App.feedback.8.docx)

**Comment by thejoefriel** [comment]
Hey @dupreesi - I think I'm right in saying that for the pre-course survey PDF it no longer should include qs 8 to 10, as these are now the ones that are included in the C5 Evaluation Questions spreadsheet instead?

So for the pre-course survey it would be:
- The extra demographic questions @elysabethc5 provided: https://docs.google.com/document/d/1ZSPrqGqbrL8FdVI7LjIKpfKFkI1BWyZPldwwzGYRv5Q/edit?usp=sharing
- Qs 3 to 7 from the PDF: https://github.com/yalla-coop/connect5/files/3157551/Pre-course.SurveyMonkey_151565181.pdf
- The pre-survey questions in the C5 Evaluation Questions spreadsheet: https://docs.google.com/spreadsheets/d/1_b_FhwKMmhh4wQpyU0i8vRh3b5cK37TvT3E5mq3FiUY/edit#gid=0

Let me know if I've misunderstood!

**Comment by dupreesi** [comment]
@thejoefriel I think so! Thanks for that

**Comment by thejoefriel** [comment]
@dupreesi sorry I'm also a bit confused by the train the trainer form? There are questions in the spreadsheet and there's also the doc? Am I right that we're just using the questions in the spreadsheet now? I imagine we'd also want to ask them the questions about their trainer as well, as this trainer feedback should be consistent across all types of sessions?

**Comment by dupreesi** [comment]
@thejoefriel yes the train the trainer pdf won't be used anymore... so only the questions in the spreadsheet...
